### ISSUE TITLE
XFCE Title bars are not the same color as menu bars

### ISSUE THREAD

**Issue by isantop** [system76/pop-gtk-theme#10]
**Reporter info**

 ```
 * Distribution -
 * Gtk+ 3.0 version -
 * Desktop environment - 
 ```

**Pop version (if you knew)**
1.0.8

**Related Application and/or Shell theming (or name of widget)**
Titlebars in XFCE

**Actual issue**
Title bars in XFCE are the wrong color. They do not blend in seamlessly with menu bars

**Expected behavior**
Menu bars and title bars are the same color, thus blending in with each other.

**Comment by isantop** [comment]
![53ebz4q](https://cloud.githubusercontent.com/assets/5883565/25729912/d8510d78-30f5-11e7-95e8-f241b443b6a9.png)

**Comment by isantop** [comment]
Fixed in 6d9562b4
